force-app/main/default/flowDefinitions/Prevent_Duplicate_Session_Regs.flowDefinition-meta.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<FlowDefinition xmlns="http://soap.sforce.com/2006/04/metadata">
```

Comment:
we dont need flow definitions. Remove its metadata and it should be removed from the package.xml. (In package.xml use flow instead of flowdefinition)